suricata.yaml.in

```diff
@@ -1857,7 +1858,9 @@ reference-config-file: @e_sysconfdir@reference.config
 ## Include other configs
 ##
 
-# Includes.  Files included here will be handled as if they were
-# inlined in this configuration file.
+# Includes:  Files included here will be handled as if they were
+# in-lined in this configuration file. Put the absolute pathname
+# of the configuration file(s) to include here. You can specify
+# more than 2 configuration files, if needed.
```

Comment:
Did relative filenames not work?

Reply:
They do work; I'll clarify in an update.